src/frontend/qt_sdl/OSD.cpp

```diff
@@ -438,6 +448,26 @@ void DrawGL(float w, float h)
     glEnable(GL_BLEND);
     glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_ALPHA);
 
+    for (auto lo = LuaScript::LuaOverlays.begin(); lo != LuaScript::LuaOverlays.end();)
+    {
+        OverlayCanvas& overlay = *lo;
+        if (overlay.isActive){
+            glGenTextures(1,&overlay.GLtexture);
```

Comment:
why are you recreating the texture for every draw when you even already have a storage place to keep it across frames?